src/Compatibility/ControlGallery/src/Core/CoreGalleryPages/ListViewCoreGalleryPage.cs

```diff
@@ -175,7 +175,7 @@ public ListViewViewModel()
 					new Employee ("Andrew 3", TimeSpan.FromDays (1000), 60),
 				};
 
-				Enumerable.Range(0, 9000).Select(e => new Employee(e.ToString(), TimeSpan.FromDays(1), 60)).ForEach(e => Employees.Add(e));
+				Enumerable.Range(0, 9000).Select(e => new Employee(e.ToString(), TimeSpan.FromDays(1), 60)).ToList().ForEach(e => Employees.Add(e));
```

Comment:
I had to move some enumerable extensions back to core. The gallery was able to use them previously because controls has IVT enabled for gallery but I didn't really want to add that to core